DeepCat.js

```diff
@@ -500,7 +500,33 @@
 	}
 
 	function hasHintCookie() {
-		return mw.cookie.get( '-deepcat-hintboxshown' ) === makeHintboxCookieToken( mw.msg( 'deepcat-hintbox-text' ) );
+		var storedData;
+		if( typeof Storage !== 'undefined' ) {
```

Comment:
Just curious, is it good practice to check this instead of checking for `localStorage`?